aws_lambda_script/main.py

```diff
@@ -0,0 +1,189 @@
+import json
+import csv
+import os
+from typing import Dict, List
+import logging
+from io import StringIO
+import boto3
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Environment variables
+SOURCE_BUCKET = os.environ.get("SOURCE_BUCKET", "sdp-dev-tech-audit-tool-api")
+SOURCE_KEY = os.environ.get("SOURCE_KEY", "new_project_data.json")
+DESTINATION_BUCKET = os.environ.get("DESTINATION_BUCKET", "sdp-dev-tech-radar")
+DESTINATION_KEY = os.environ.get("DESTINATION_KEY", "onsTechDataAdoption.csv")
+
+
+def get_json_from_s3(bucket: str, key: str) -> Dict:
+    """Retrieve JSON data from S3 bucket"""
+    try:
+        s3_client = boto3.client("s3")
```

Comment:
I'd move this definition to global scope or else it'll make a new instance each time you call the method. If you define `s3_client` in global scope, you can also reuse it in other parts such as on L89.

Reply:
Moved to global scope and renamed to 'client'.